system/ee/EllisLab/ExpressionEngine/Service/Validation/Rule/MaxLength.php

```diff
@@ -28,12 +31,8 @@ public function validate($key, $value)
 			return FALSE;
 		}
 
-		if (function_exists('mb_strlen'))
-		{
-			return (mb_strlen($value) > $length) ? FALSE : TRUE;
-		}
+		return (ee_mb_strlen($value) < $length) ? true : false;
```

Comment:
This doesn't follow the style guide for capital true/false values. Also looks like it was changed from `>` to `<` but if you flip the conditional, you'd need it to be `<=` so it includes the total length. Could also be shortened to `return (ee_mb_strlen($value) <= $length);`

Reply:
> This doesn't follow the style guide for capital true/false values

IMO EE should move away from that and move towards PSR standards. EE was originally written before PSR was a thing, now that it is an industry standard EE should use it.